### PR TITLE
feat(oma-refresh): improve verify error message ...

### PR DESCRIPTION
**Pull request by eatradish**
... also fix progress event not clear

### CODE REVIEW

**Comment by eatradish** [comment]
图片

**Comment by eatradish** [comment]
v2:

图片

File: i18n/en-US/oma.ftl [hunk 1]
```diff
@@ -236,3 +236,4 @@ tum-1 = oma has found { $updates } updates available for your system.
 tum-1-with-security =  oma has found { $updates } updates available for your system ({ $security } security updates are marked in { $security_str }).
 tum-2 = In order to fulfill the operations you specified, oma will need to perform the following changes to your system components. The details are as follows.Details of the component changes for this operation are as follows.
 security = security update(s)
+verify-error = An error was encountered while verifying the signature of { $p }.
```
Comment (MingcongBai) [inline i18n/en-US/oma.ftl:239]:
```
An error occured while verifying the signature for { $p }.
```

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: i18n/en-US/oma.ftl (hunk 1 above)
Comment (eatradish) [reply i18n/en-US/oma.ftl]: Done

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**